omod/src/main/webapp/pages/scheduleProviders.gsp

```diff
@@ -23,7 +23,7 @@
 
     ui.includeJavascript("appointmentschedulingui", "app.js")
     ui.includeJavascript("appointmentschedulingui", "controllers/selectMultipleAppointmentTypesController.js")
-    ui.includeJavascript("appointmentschedulingui", "controllers/scheduleProvidersController.js")
+    /////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
What is this additional here?

Reply:
Ooooohhh, I'm veryyy sorryyyyyy. Just forget removing that.

Reply:
Ooohh, I'm very sorry for such mistakes. I just Forgot to remove it.

Reply:
No worries!  Go ahead and remove it when you get a chance.